source/crud/read-operations/cursor.txt

```diff
@@ -7,10 +7,11 @@ Access Data From a Cursor
 Read operations that return multiple documents do not immediately return
 all values matching the query. Because a query can potentially match
 very large sets of documents, these operations rely upon an
-object called a **cursor** that fetches documents in batches to reduce
-both memory consumption and network bandwidth usage. Cursors
-are highly configurable and offer multiple interaction paradigms for
-different use cases.
+object called a `cursor
+<https://mongodb.github.io/node-mongodb-native/3.4/api/Cursor.html>`_
+that fetches documents in batches to reduce both memory consumption and
+network bandwidth usage. Cursors are highly configurable and offer
+multiple interaction paradigms for different use cases.
```

Comment:
When using the adjectives "multiple ___ for different ____" together, it sounds like you're describing a cross-product. I think you only need one adjective. E.g. "...offer interaction paradigms for different use cases" or "offer multiple interaction paradigms." If it is actually a cross-product, feel free to leave as is.

Reply:
Agreed. I think it does apply as a cross product here -- each use case can traverse a cursor in a few different ways.